shell/platform/windows/win32_dpi_helper.cc

```diff
@@ -43,6 +43,20 @@ Win32DpiHelper::~Win32DpiHelper() {
   }
 }
 
+void Win32DpiHelper::SetDpiAwerenessAllVersions() {
+  if (permonitorv2_supported_) {
+    SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
+  } else {
+    std::cerr << "Per Monitor V2 DPI awareness is not supported on this "
+                 "version of Windows. Setting System DPI awareness\n";
+    AssignProcAddress(user32_module_, "GetDpiForSystem", get_dpi_for_system_);
+    AssignProcAddress(user32_module_, "SetProcessDpiAware",
+                      set_process_dpi_aware_);
+
+    SetProcessDPIAware();
```

Comment:
Whilst it’s tempting to do this I do not recommend it.  The host app should be in charge of it’s DPI mode, not flutter.  For add to app scenarios you’ll have to support all DPI modes in the fullness of time.  Also, it’s not recommended to set DPI programatically, that should be done in the calling app’s manifest.  Recommend updating this to sniff the current DPI mode and act accordingly.  You’ll have to put this logic in testbed / example tho

Reply:
From what I understand, setting the awareness in the application manifest will override this calls anyway. I added the calls to ensure that even if the host didn't provide it in the manifest, we would. I tested with Testbed, by setting a different awareness in the manifest and it took that value over the api call. What do you think?